### ISSUE TITLE
The input device won't register

### ISSUE THREAD

**Issue by abi-aryan** [constcharptr/lyrebird#50]
So, I chose a pre-set and toggled the record button

![app](https://user-images.githubusercontent.com/7281944/104319246-9e5d2000-5506-11eb-855d-83d24e7d70af.png)

I am using PulseAudio Volume Control and not able to select the Lyrebird Input Device

![input](https://user-images.githubusercontent.com/7281944/104319585-13c8f080-5507-11eb-809b-4d6886f9e7dd.png)

In the system settings as well, the input does show but nothing gets recorded. You can see no amplification in there.

![input](https://user-images.githubusercontent.com/7281944/104319836-75895a80-5507-11eb-9dee-154c241e4f19.png)

Also, I am not sure where the output is supposed to come but I am pretty sure it should be on the system itself if only the input device was able to record something. FYI: I chose Speakers Built in Audio for the output.

**Comment by harrego** [comment]
If you hear no audio could you first confirm that Lyrebird has your correct input mic selected. Go back to the "Recording" tab in pavucontrol, make sure you have "All Streams" selected at the bottom of the window, and check that SoX is using the correct device. If it isn't you can click to change it.

![Screenshot from 2021-01-12 13-39-49](https://user-images.githubusercontent.com/41309736/104322025-4a3d4600-54dc-11eb-94c8-b934d6acfc72.png)

**Comment by abi-aryan** [comment]
yes, it is although on my end it shows **remapped stream from** instead of **SoX record from**

![1](https://user-images.githubusercontent.com/7281944/104998718-b296a500-5a51-11eb-95de-de3301631326.png)

**Comment by abi-aryan** [comment]
I think I fixed it. 

Here's what I tried doing:

1. check if sox, pyaudio and PulseAudio are installed - [ all were!]
2. restart my device and that fixed the null input/output thing and in the system control it now gave me the Lyrebird Input/Output - there I chose Build-In Stereo Input as my standard input
3. Since sox was not able to handle the file type so I extended sox file types by choosing 
`sudo apt install libsox-fmt-all`
4. From Playback Tab I chose System Built-In for Output instead of Lyrebird Output
![1](https://user-images.githubusercontent.com/7281944/105003894-3dc76900-5a59-11eb-9d9e-15e57dc71aa5.png)
